### DISCUSSION TITLE
Rename some variables

### DISCUSSION THREAD

**Discussion by fidojones** [blackears/cyclopsLevelBuilder#131] [General]
I added the addon to my project, but some variables in the numeric_line_edit.gd interfere with my State machine variables

enum State{ IDLE, READY, DRAGGING, TEXT_EDIT }
var state:State = State.IDLE

I had to rename both, to prevent errors with the addon. Could be posible to change this names in he oficial plugin to avoid posible future errors?

**Comment by blackears** [comment]
I've pushed an update that changes the enum name.  You'll need to get the update from the latest dev branch.
